test/lib/rules/yield-effects-spec.js

```diff
@@ -25,9 +79,73 @@ ruleTester.run("yield-effects", rule, {
   ],
   invalid: [
     {
+      code: "import { take } from 'redux-saga';"
+          + "export default function* saga() {"
+          + "  const sagaTakeEffect = take();"
+          + "  yield sagaTakeEffect;"
+          + "}",
+      errors: [ERRORS.noyield]
+    },
+    {
+      code: "import { take as sagaTake } from 'redux-saga';"
+          + "export default function* saga() {"
+          + "  const sagaTakeEffect = sagaTake();"
+          + "  yield sagaTakeEffect;"
```

Comment:
Ok, I agree. So which additional tests / checks you suggesting to complete this rule?